docs/src/man/guide.md

```diff
@@ -14,6 +14,17 @@ From the Julia REPL, type `]` to enter the Pkg REPL mode and run
 pkg> add Documenter
 ```
 
+It is generally recommended to install Documenter into a documentation-specific project stored
+in the `/docs` subdirectory of your package. To do this, navigate to your package's root folder
+and do
+
+```
+pkg> activate docs
+
+(docs) pkg> add Documenter
+```
+
+This will create `Project.toml` and `Manifest.toml` files in the `/docs` subdirectory. 
```

Comment:
```suggestion
This will create `Project.toml` and `Manifest.toml` files in the `docs/` subdirectory. 

Note that for packages, you also likely need to have your package that you are documenting as a  ["dev dependency"](https://pkgdocs.julialang.org/v1/managing-packages/#developing) of the `docs/` environment.

See also [the Pkg.jl documentation on working with project environments](https://pkgdocs.julialang.org/v1/environments/).
```